src/cmd/strip.go

```diff
@@ -1,22 +1,19 @@
 package main
 
 import (
-	"flag"
-	"fmt"
-	"os"
 	"io"
-	"strings"
+	"os"
 )
 
-func main(){
-	var path string
-	flag.StringVar(&path, "i", "", "PATH to source file")
-	flag.Parse()
-	fmt.Println(path)
-	dest := strings.Replace(path, "./", "", -1)
-	dest = strings.Replace(dest, "/", ".", -1)
-	copyFile(path, "../../test/" + dest)
-}
+//func main(){
```

Comment:
why this commented out ?

Reply:
All this is being treated as a single package and there are two calls to main, and as far as I check the main one is the main one of cgogen.go

Reply:
If not needed then delete it instead of commenting it out